src/model/a320-interior/A320_NEO_INTERIOR_LOD00.gltf

```diff
@@ -187154,6 +187154,44 @@
             ],
             "mesh": 1382,
             "name": "GEAR_EXTN_Decal"
+        },
+        {
+            "translation": [
+                0.5676975,
+                1.2626953,
+                10.8251953
+            ],
+            "rotation": [
+                0,
+                0,
+                0,
+                1
+            ],
+            "scale": [
+                1,
+                1,
+                1
+            ],
+            "name": "PILOT_0"
+        },
+        {
+            "translation": [
+                -0.5676975,
+                1.2626953,
+                10.8251953
+            ],
+            "rotation": [
+                0,
+                0,
+                0,
+                1
+            ],
+            "scale": [
+                1,
+                1,
+                1
+            ],
+            "name": "PILOT_1"
```

Comment:
There is no way to add nodes only to LOD00, i tried through src/models, but it will add to all other LOD's where this nodes is already exist. Better way if possible to add in build.js option with selection add modification to all lods or only selected.

Anyway i can add this by a hand (every time when Asobo changed gltf file)

Reply:
> It would be better if we don't make changes directly to this file, that way we don't have to redo them when Asobo updates the model.

Reworked. Block. Added script with adding pilots in LOD00. GLTF revert back to default.